tests/suite/cli_exact.rs

```diff
@@ -699,7 +699,7 @@ async fn undefined_linked_toolchain() {
         .expect_err_ex(
             &["cargo", "+bogus", "test"],
             r"",
-            "error: toolchain 'bogus' is not installed\n",
+            "error: toolchain 'bogus' is not installed.\n",
```

Comment:
Now considering adding a case which shows the second line, for example the one in the original issue.